pkg/store/employee/employee.go

```diff
@@ -28,7 +29,23 @@ func (s *store) OneByTeamEmail(teamEmail string) (*model.Employee, error) {
 	return employee, s.db.Where("team_email = ?", teamEmail).First(&employee).Error
 }
 
-func (s *store) All() ([]*model.Employee, error) {
+// Search get employees by query and pagination
+func (s *store) Search(query request.GetListEmployeeQuery) ([]*model.Employee, int64, error) {
+	db := s.db.Table("employees")
+	var total int64
+
+	if query.WorkingStatus != "" {
+		db = db.Where("working_status = ?", query.WorkingStatus)
+	}
+	db = db.Count(&total)
+
+	if query.Page > 1 && query.Size >= 1 {
+		db = db.Offset(int((query.Page - 1) * query.Size))
```

Comment:
I think we need .Limit to get only "size" records

Reply:
oke